### PR TITLE
Ajustes

### PR DESCRIPTION
**Pull request by relferreira**

### CODE REVIEW

**Comment by relferreira2** [comment]
test

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Test
 
-hell 3o
+hell 4o
```
Comment (relferreira2) [inline README.md]: are you sure?

Comment (relferreira2) [inline README.md]: Thats ok

Comment (relferreira) [inline README.md]: now I'm not sure

Comment (relferreira2) [inline README.md]: What?

Comment (relferreira2) [inline README.md]: ?

**Pull request merged**